pkg/cmd/issue/view/view.go

```diff
@@ -76,20 +79,37 @@ func viewRun(opts *ViewOptions) error {
 	}
 	apiClient := api.NewClientFromHTTP(httpClient)
 
-	issue, _, err := issueShared.IssueFromArg(apiClient, opts.BaseRepo, opts.SelectorArg)
+	issue, repo, err := issueShared.IssueFromArg(apiClient, opts.BaseRepo, opts.SelectorArg)
 	if err != nil {
 		return err
 	}
 
-	openURL := issue.URL
-
 	if opts.WebMode {
+		openURL := issue.URL
 		if opts.IO.IsStdoutTTY() {
 			fmt.Fprintf(opts.IO.ErrOut, "Opening %s in your browser.\n", utils.DisplayURL(openURL))
 		}
 		return utils.OpenInBrowser(openURL)
 	}
 
+	if opts.Comments {
+		var s *spinner.Spinner
+		if opts.IO.IsStdoutTTY() {
+			s = utils.Spinner(opts.IO.ErrOut)
+			utils.StartSpinner(s)
```

Comment:
Missed this on my previous review: the IO object has methods to start/stop the progress indicator; you do not have to interface with spinner directly nor check `IsStdoutTTY` 👌